ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py

```diff
@@ -248,6 +248,53 @@ def test_linear(
     )
 
 
+# alpha_dropout
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.common.alpha_dropout",
+    d_type_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=1,
+        shared_dtype=True,
+        min_value=2,
+        max_value=5,
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=2,
+    ),
+    p=st.floats(min_value=0.0, max_value=1.0),
+    axis=st.integers(min_value=0, max_value=1),
+    training=st.booleans(),
+    mode=st.sampled_from(["upscale_in_train", "downscale_in_infer"]),
```

Comment:
This mode wasn't indicated in the docs we need to test function based on Documentation
https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/functional/alpha_dropout_en.html#alpha-dropout